### PR TITLE
[BRE-101] Remove dept-devops from CODEOWNERS

### PR DESCRIPTION
**Pull request by michalchecinski**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

https://bitwarden.atlassian.net/browse/BRE-101

Remove `dept-devops` from CODEOWNERS and add `dept-bre` for `.github/workflows/release.yml`.

## Code changes

* **.github/CODEOWNERS:** Remove `dept-devops` and add `dept-bre` for `.github/workflows/release.yml`.

## Before you submit
- Please check for formatting errors (`dotnet format --verify-no-changes`) (required)
- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -43,3 +34,9 @@ store/google/en
 
 ## Utils ##
 store/google/Publisher
+
+## These are shared workflows ##
+.github/workflows/cleanup-rc-branch.yml
+.github/workflows/release.yml
+.github/workflows/version-auto-bump.yml
+.github/workflows/version-bump.yml
```
Comment (vgrassia) [inline .github/CODEOWNERS:42]:
```suggestion
## These workflows have joint ownership ##
.github/workflows/build.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
.github/workflows/build-beta.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
.github/workflows/cleanup-rc-branch.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
.github/workflows/release.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
.github/workflows/version-auto-bump.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
.github/workflows/version-bump.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -7,15 +7,6 @@
...
 
-# DevOps for Actions and other workflow changes
-.github/workflows @bitwarden/dept-devops
-
-# DevOps for Version Bumping
-src/App/Platforms/Android/AndroidManifest.xml
-src/iOS.Autofill/Info.plist
-src/iOS.Extension/Info.plist
-src/iOS.ShareExtension/Info.plist
-src/App/Platforms/iOS/Info.plist
```
Comment (vgrassia) [inline .github/CODEOWNERS:18]: Please add these back in. The comment can be updated to read `# Shared ownership for version bump automation`

**Review by vgrassia — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -43,3 +34,18 @@ store/google/en
...
+.github/workflows/release.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+.github/workflows/version-auto-bump.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+.github/workflows/version-bump.yml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+
+# Shared ownership for version bump automation
+src/App/Platforms/Android/AndroidManifest.xml @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/iOS.Autofill/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/iOS.Extension/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/iOS.ShareExtension/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
+src/App/Platforms/iOS/Info.plist @bitwarden/dept-bre @bitwarden/dept-development-mobile
```
Comment (vgrassia) [inline .github/CODEOWNERS:51]:
```suggestion
src/App/Platforms/Android/AndroidManifest.xml
src/iOS.Autofill/Info.plist
src/iOS.Extension/Info.plist
src/iOS.ShareExtension/Info.plist
src/App/Platforms/iOS/Info.plist
```

These have to have no owner in order for our automatic PR process to work between the bitwarden-devops-bot and the github-bot.  Sorry, I wasn't clear in my first message.

Comment (michalchecinski) [reply .github/CODEOWNERS:51]: Ah, gotcha. Thanks!

**Review by vgrassia — APPROVED** [review]

**Review by fedemkr — APPROVED** [review]

**Pull request merged**